app/controllers/lakeshore/ingest_controller.rb

```diff
@@ -48,7 +48,7 @@ def validate_duplicate_upload
       end
 
       def validate_preferred_representations
-        return unless represented_resources.present?
+        return if force_preferred_representation? || represented_resources.empty?
```

Comment:
@RudyOnRails I find the guard clauses keep the logic contained in the method, instead of separating logic in the filter and then in the action in the method. It also keeps the pattern consistent. Each before filter uses a guard clause. Since the logic varies per method, you'd need to make separate `skip_before_filter` declarations for each method, each with an if clause and an only clause. To me, that increases the cognitive load of each of those lines, requiring the reader to keep more in their head until they get the actual method call. By using just `before_filter`, the only responsibility of that line is to call the method for specific actions. This reduces the complexity of that one line. Then, the logic regarding what the actual method does is relegated to the method itself, and not split between the method and its filter invocation.

Reply:
I see your point. We would need to add two more `skip_before_action`'s each with a (new & separate) guard clause method, just to make the validation guts simpler.

Maybe one day all this model validation logic (`before_filter`s and 4+ methods) will be in the model, and we can call `.save`'s on the `Lakeshore::Ingest` object, firing off model validations, adhering to skinny-controllers/fat-models.